src/components/Nav/index.tsx

```diff
@@ -339,24 +334,24 @@ const Nav: React.FC<IProps> = ({ path }) => {
 
   const ednLinks: Array<IItem> = [
     {
-      text: "home",
+      text: t("home"),
```

Comment:
Line 433 below, we map this as a key... similar to a note elsewhere, should we remove the use of `<Tranlation/>` below?
```ts
          {ednLinks.map((link, idx) => (
            <NavLink
              key={idx}
              to={link.to}
              isPartiallyActive={link.isPartiallyActive}
            >
              <Translation id={link.text} />
            </NavLink>
          ))}
```

Reply:
Yes, same case as the prev comment. Thanks.